dashboard/test/ui/runner.rb

```diff
@@ -234,10 +235,16 @@ def log_browser_error(msg)
   puts msg if $options.verbose
 end
 
-def run_tests(env, arguments, log_prefix)
+def run_tests(env, feature, arguments, log_prefix)
   start_time = Time.now
-  puts "#{log_prefix}cucumber #{arguments}"
-  Open3.popen3(env, "cucumber #{arguments}") do |stdin, stdout, stderr, wait_thr|
+  scenario_count = ParallelTests::Cucumber::Scenarios.all([feature], test_options: arguments).length
+  if scenario_count.zero?
+    return true, '0 scenarios', '', Time.now - start_time
+  end
```

Comment:
This is great - we should also apply this to our test status page ([starting here](https://github.com/code-dot-org/code-dot-org/blob/v2017-04-27.0/dashboard/test/ui/runner.rb#L302-L303)) which currently includes rows for tests that will not run any scenarios.  It's most noticeable on [the eyes status page](https://test-studio.code.org/ui_test/test_status_Eyes.html) where we include rows for all 27 eyes-related features on iPhone even though only one of them uses the `@eyes_mobile` tag and will actually run anything.